pinot-api/src/test/java/org/apache/pinot/client/QueryClientTest.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.client;
+
+import org.testng.annotations.Test;
+
+
+/**
+ *  Tests for resp body java client api coding utf-8
+ */
+public class QueryClientTest {
+  @Test
+  public void ClientRespCoding() {
+    Connection connection =
+        ConnectionFactory.fromHostList("10.85.126.103:8099", "10.85.126.104:8099", "10.85.126.105:8099");
```

Comment:
This is not a unit test. You need to setup the cluster in these ip addresses in order to make it work...

Reply:
yes, If  want to do unit tests,I need to launch zookeeper, controller, broker, and server first, then create schema, table, upload data, and finally do query tests. It is a little complicated, so I follow ConnectionFactoryTest and encapsulate the tests into the main function